scripts/local_dev.sh

```
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# スクリプトのディレクトリを取得
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_ROOT="$( cd "$SCRIPT_DIR/.." && pwd )"
+
+# プロジェクトのルートディレクトリに移動
+cd "$PROJECT_ROOT"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)